src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.ts

```diff
@@ -231,6 +235,47 @@ export class ProgrammingExerciseDetailComponent implements OnInit, OnDestroy {
         });
     }
 
+    /**
+     * Parse tasks and corresponding test cases from problem statement for this exercise
+     */
+    extractTasksFromProblemStatement(): void {
+        this.programmingExerciseService.extractTasksFromProblemStatement(this.programmingExercise.id!).subscribe({
```

Comment:
I think this has to be changed now as the endpoint was removed
=> simple exercise update with no changed values